reactive/k8s.py

```diff
@@ -31,11 +32,29 @@ def config_changed():
         docker_compose_kill_remove('proxy')
         remove_state('proxy.available')
 
+@when('kubelet.available', 'proxy.available', 'cadvisor.available')
+def final_messaging():
+    ''' Lower layers emit messages, and if we do not clear the status messaging
+        queue, we are left with whatever the last method call sets status to.
         if config.changed('version'):
```

Comment:
This if statement looks out of place in this comment, unless you are specifically commenting on that.